cmd/gossamer/flags.go

```diff
@@ -285,6 +285,14 @@ var (
 		Usage: "Retain number of block from latest block while pruning",
 		Value: 256,
 	}
+
+	// Pruning triggers the online pruning of historical state tries. It's either full or archive. To enable pruning the value
+	// should be set to `full`.
+	Pruning = cli.StringFlag{
+		Name:  "pruning",
+		Usage: `State trie online pruning ("full", "archive")`,
+		Value: "full",
```

Comment:
wondering if we should leave this as archive for now until we test the changes more thoroughly

Reply:
I feel if we default to archive then pruning wouldn't be tested enough.